From e78b1f905a016c3652b5b62a1243b6d51bc2558f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ernst=20Sj=C3=B6strand?= <ernst@stickybit.se>
Date: Wed, 8 Feb 2017 16:13:52 +0100
Subject: [PATCH 1/3] build.sh: Support our build

Allow us to fake Ubuntu RID to pass all build steps.
Also allow more recent clang versions, so we can use the
same clang across all projects.
---
 src/corehost/build.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/corehost/build.sh b/src/corehost/build.sh
index 9da178a..e3f7552 100755
--- a/src/corehost/build.sh
+++ b/src/corehost/build.sh
@@ -164,7 +164,7 @@ esac
 fi

 # __rid_plat is the base RID that corehost is shipped for, effectively, the name of the folder in "runtimes/{__rid_plat}/native/" inside the nupkgs.
-__rid_plat=
+__rid_plat="linux"
 init_rid_plat
 
 if [ -z $__rid_plat ]; then
@@ -208,6 +208,9 @@ if [ $__CrossBuild == 1 ]; then
     if command -v "clang-3.9" > /dev/null 2>&1; then
         export CC="$(command -v clang-3.9)"
         export CXX="$(command -v clang++-3.9)"
+    elif command -v clang > /dev/null 2>&1; then
+        export CC="$(command -v clang)"
+        export CXX="$(command -v clang++)"
     else
         echo "Unable to find Clang 3.9 Compiler"
         echo "Install clang-3.9 for cross compilation"
-- 
2.9.3

